tests/versioning_test.go

```diff
@@ -1535,16 +1564,33 @@ func (s *versioningIntegSuite) TestWorkflowGetsStuckWhenDisablingLoadingUserData
 
 	run, err := s.sdkClient.ExecuteWorkflow(ctx, sdkclient.StartWorkflowOptions{
 		TaskQueue:                tq,
-		WorkflowExecutionTimeout: 5 * time.Second,
+		WorkflowExecutionTimeout: 10 * time.Second,
 	}, "wf")
 	s.Require().NoError(err)
-	err = run.Get(ctx, nil)
-	var timeoutError *temporal.TimeoutError
-	s.Require().ErrorAs(err, &timeoutError)
+
+	// should not run on versioned worker
+	time.Sleep(2 * time.Second)
 	s.Require().Equal(int32(0), runs.Load())
+
+	wrk.Stop()
+
+	// start unversioned worker and let task run there
+	wrk2 := worker.New(s.sdkClient, tq, worker.Options{
+		BuildID:                          "v1",
+		UseBuildIDForVersioning:          false,
```

Comment:
Seems irrelevant

Reply:
I thought "use versioning: false" would be nice to be explicit. I can remove